### PR TITLE
Add solutions cards

### PR DESCRIPTION
**Pull request by spuigmitjadxc**

### CODE REVIEW

File: app/assets/config/decidim_challenges_manifest.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
 //= link decidim/challenges/add_challenge.js
 //= link decidim/challenges/admin/challenges_form.js
-//= link_directory ../stylesheets/decidim/challenges .css
\ No newline at end of file
+//= link_directory ../stylesheets/decidim/challenges .css
```
Comment (Leusev) [inline app/assets/config/decidim_challenges_manifest.js:3]: Is this space between `challenges` and `.css` left on purpose?
Same with `solutions` and `.css`

File: app/cells/decidim/solutions/solution_m/show.erb [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            <div class="card_text--sdg--image">
+                <div class="ods">
+                    <% if resource_sdg %>
+                        <div class="text-container">
+                            <p><%= t(resource_sdg + ".logo.line1", scope: "decidim.components.sdgs") %></p>
+                            <p><%= t(resource_sdg + ".logo.line2", scope: "decidim.components.sdgs") %></p>
+                        </div>
+                    <% end %>
+                    <% if resource_sdg_index %>
+                        <img alt="Logo SDG <%= resource_sdg_index%>" src="/assets/decidim/sdgs/ods-<%= resource_sdg_index%>.svg" class="solution--view">
```
Comment (Leusev) [inline app/cells/decidim/solutions/solution_m/show.erb:41]: Maybe it would be better using a translation tag than harcoded text here

**Review by Leusev — CHANGES_REQUESTED** [review]
Looks pretty good, just a couple of little things commented on :wink: 

1. Check rubocop lints please
image

2. Same here, when PR #7 is reviewed/merged with master, you'll could do a merge from master here in order to fix failing tests

Thanks!! :smile:

**Comment by tramuntanal** [comment]
PR #7 has already been merged @spuigmitjadxc 

Can you also resolve merge conflicts and check that all specs related to solutions cards succeed please?

**Review by Leusev — CHANGES_REQUESTED** [review]
We already have merged PRs #11  #13 and  #15, do master merge please :smile:

**Comment by Leusev** [comment]
Hi @spuigmitjadxc 
can you merge master with this PR please? 
Thanks!

**Comment by Leusev** [comment]
Hi @spuigmitjadxc 
same here, merge with master please. Furthermore could you check Lint tests please?
Thanks!

**Pull request merged**